packages/expo-image/src/ExpoImage.web.tsx

```diff
@@ -1,8 +1,108 @@
 import React from 'react';
-import { Image } from 'react-native';
 
-export default function ExpoImage({ source, ...props }) {
-  const resolvedSource = source ?? {};
+import { ImageContentFit, ImageContentPosition, ImageProps, ImageSource } from './Image.types';
+import { resolveContentFit, resolveContentPosition } from './utils';
 
-  return <Image {...props} source={resolvedSource} />;
+const resolveAssetSource = (source: ImageSource | string | number | undefined | null) => {
+  if (source === null || source === undefined) return undefined;
+
+  if (typeof source === 'string') {
+    return { uri: source };
+  }
+  if (typeof source === 'number') {
+    return { uri: String(source) };
+  }
+
+  return source;
+};
+
+const getObjectFitFromContentFit = (contentFit?: ImageContentFit) => {
```

Comment:
I think you can just use the enum, it has the same values so no need to map them 😉